doc/rst/users/build.rst

```diff
@@ -34,9 +46,71 @@ package manager (https://github.com/spack/spack).
 SCR and all of its dependencies exist in a Spack package. After downloading
 Spack, simply type::
 
-  spack install scr
+  spack compiler find --scope site
+  spack install scr@develop resource_manager=SLURM
+
+This will install the component libraries, DTCMP, LWGRP, libyogrt, and pdsh packages (and possibly an MPI and a
+C compiler if needed).
+
+By default, Spack will attempt to build all dependencies for SCR, including many packages such as MPI and OpenSSL that are already installed on many systems. Often it is preferrable to use the pre-installed versions. Spack uses file called 'packages.yaml' to tell it where to find external packages. Information on setting up 'packages.yaml' is available in the Spack documentation (see https://spack.readthedocs.io/en/latest/configuration.html). Spack searches for the packages.yaml file in several locations. When installing Spack in user space, customizations can be placed in $SPACK/etc/packages.yaml or $HOME/.spack/packages.yaml.
+
+Here is an example of a simple `packages.yaml` file::
+
+    packages:
+      all:
+        providers:
+          mpi: [mvapich2,openmpi,spectrum-mpi]
+
+      openmpi:
+        buildable: false
+	- spec: openmpi/4.1.0
+	  prefix: /usr/tce/packages/openmpi/openmpi-4.1.0
+
+      mvapich2:
+        buildable: false
+	externals:
+	- spec: mvapich2
+        modules:
+        - mvapich2
+
+      spectrum-mpi:
+        buildable: false
+	externals:
+	- spec: spectrum-mpi@rolling-release%xl_r
+	  prefix: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-xl-2020.03.18/
+	- spec: spectrum-mpi@rolling-release%xl
+	  prefix: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-xl-2020.03.18/
```

Comment:
Using path which looks to be the default for a "root install" from https://www.ibm.com/docs/en/SSZTET_10.2/install/smpi_install_pdf.pdf
```suggestion
	- spec: spectrum-mpi
	  prefix: /opt/ibm/spectrum-mpi
```

Reply:
The path is nicer, but I think I'm using the %xl_r to pull in the correct MPI as a side-effect. I'll commit it, but try to check it on Lassen.